locksmith/src/models/stripeCustomer.ts

```diff
@@ -0,0 +1,11 @@
+import { Table, Column, Model } from 'sequelize-typescript'
+
+@Table({ tableName: 'StripeCustomer', timestamps: true })
```

Comment:
I am not sure what is the scheme we currently have in place in the app, but this needs to match the value used in the migration

Reply:
Doh. Sorry. thanks for catching it!